schema.sql

```diff
@@ -15,4 +15,37 @@ CREATE TABLE animals (
 
 -- Milestone: Vet clinic database: query and update animals table
 
-ALTER TABLE animals ADD species VARCHAR(100);
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR(100);
+
+-- Vet clinic database: query multiple tables
+
+-- Create a table named owners with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- full_name: string
+-- age: integer
+
+CREATE TABLE owners (
+  id INT UNIQUE GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  full_name VARCHAR(100),
+  age INT
+);
+
+-- Create a table named species with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- name: string
+
+CREATE TABLE species (
+  id INT UNIQUE GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(100)
+);
```

Comment:
* Kindly add a primary key that increases automatically as new records are created (ie PRIMARY KEY (id)) [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=the%20following%20columns%3A-,id%3A%20integer%20(set%20it%20as%20autoincremented%20PRIMARY%20KEY),-full_name%3A%20string)

Reply:
Thank you for pointing it out